src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -27,6 +27,7 @@
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
     <DevelopmentDependency>true</DevelopmentDependency>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <NoPublicAPIs>true</NoPublicAPIs>
```

Comment:
The model here is a bit weird.

Would we want to model this as a <PublicAPI> property...perhaps a tri state?
Having a different property for multi-tfm APIs seems odd.
Seems like we should opt in to <PublicAPI>, not opt out, for new projects? (could set the default more widely with common.props?)

not sure of pros/cons.

Reply:
Thanks for the suggestion @rrelyea . I just pushed changes. I think it reads better now, even if it is more lines than before.